package/src/components/StepsIndicator.tsx

```diff
@@ -44,6 +44,7 @@ export const StepsIndicator = ({
             <View style={styles.stepIndicatorElement}>
               <SliderTrackMark
                 key={`${index}-SliderTrackMark`}
+                value={i}
```

Comment:
`i` is the number of step and is constant for each. For having the current value on which the thumb is, `currentValue` should be used here.